test/e2e/helpers_test.go

```diff
@@ -30,9 +30,6 @@ var (
 	//go:embed resources/testdata/fleet-capi-test.yaml
 	fleetCAPITestdata []byte
 
-	//go:embed resources/config/capi-providers-secret.yaml
-	capiProvidersSecret []byte
```

Comment:
Why is this removed? This secret is a “patch” on top of existing resource as well, and is used for providing additional env depending on type of test. Like in e2e for azure PR, it adds all azure env secrets.

Reply:
It was removed just because I did not see it being used other than in the `Adding CAPI variables secret` section of [initRancherTurtles](https://github.com/rancher-sandbox/rancher-turtles/blob/ed9b78bb4cd64dab12dee4feb5dac19cab538ced/test/e2e/e2e_suite_test.go#L249), which is no longer required, but I have no problem in keeping it if it's needed.

Reply:
For this to be no longer required a secret content should be fully customizable. Right now you can only set 2 feature flags.

Reply:
If we give the user the possibility to use their own secret, do we still need to let them customize the content of the "default" secret?

Reply:
This functionality used in e2e tests. There is a PR implementing test case for [azure](https://github.com/rancher-sandbox/rancher-turtles/pull/133), it was opened a couple days prior to that. This is where the secret is used and extended.